src/mono/wasm/templates/templates/console/main.mjs

```diff
@@ -1,5 +1,16 @@
 import { App } from './app-support.mjs'
 
 App.main = async function (applicationArguments) {
+
+    App.IMPORTS.node = {
+        process : {
+            version: () => globalThis.process.version
+        }
+    };
+
+    const exports = await App.MONO.mono_wasm_get_assembly_exports("console.0.dll");
+    const text = exports.MyClass.Greeting();
+    console.log(text);
+
     await App.MONO.mono_run_main("console.0.dll", applicationArguments);
```

Comment:
This should return the exit code, so we can exit node with that. And it would cause the test to fail right there, instead of failing when checking the output.

Reply:
Radek noticed yesterday that the Program.cs file is being overwritten in the WasmTemplateTests.cs and I managed to solve the failing console tests locally by making the Program.cs file contain both the code needed for the test and the actual code needed for the wasm app to work.

Reply:
> This should return the exit code, so we can exit node with that. And it would cause the test to fail right there, instead of failing when checking the output.

sounds good. let's fix it in following PR